kolena/workflow/visualization/_activation_map.py

```diff
@@ -56,7 +56,7 @@ def blue(self, intensity: np.uint8) -> np.uint8:
         """
         raise NotImplementedError
 
-    def alpha(self, intensity: np.uint8) -> np.uint8:
+    def alpha(self, intensity: np.uint8) -> Union[np.uint8, int]:
```

Comment:
Should this just be `int` (since it's a superset of `np.uint8`)?

Reply:
Updated. This file may need another look at some point (types are off, boundary between normal int/float and uint8 is not handled well) but passes mypy and unit tests for now.